core/modules/replica/Configuration.h

```diff
@@ -544,11 +544,14 @@ class Configuration {
      * @param isDirectorTable An (optional) flag indicating if this is the "director"
      *   table of the catalog. Note there could be only one such table in a catalog,
      *   and this table must be "partitioned".
-     * @param directorTableKey The (optional) name of a column in the "director"
-     *   table which is used as an "object" identifier for astronomical objects.
-     *   This parameter applies to the director tables only. And if provided (and
-     *   allowed) the column must be found among the names of columns in a value
-     *   of parameter "columns".
+     * @param directorTableKey The (optional) name of a column representing object identifiers.
```

Comment:
`directorTableKey` is also confusing, if this is a column name then it's probably better to call it a column, "key" has some meaning in the database context which does not help here.